app/workers/email_held_bike_notification_worker.rb

```diff
@@ -0,0 +1,27 @@
+class EmailHeldBikeNotificationWorker < ScheduledWorker
+  def self.frequency
+    24.hours
+  end
+
+  def perform
+    record_scheduler_started
+    notify_of_bike_index_held
+    record_scheduler_finished
+  end
+
+  def notify_of_bike_index_held
+    Bike.held_with_match.each do |bike, match|
+      next if CustomerContact.held_bike_notification.exists?(bike_id: bike.id, user_email: bike.owner_email)
+
+      email = CustomerMailer.held_bike_email(bike, match)
+      contact = CustomerContact.build_held_bike_notification(
+        bike: bike,
+        subject: email.subject,
+        body: email.text_part.to_s,
+        sender: email.from.first,
+      )
+
+      email.deliver_now if contact.save
```

Comment:
This needs to implement the check whether the user wants to receive notifications about the bike or not - https://github.com/bikeindex/bike_index/blob/master/app/workers/email_stolen_bike_alert_worker.rb#L7

I think it makes sense to put that logic into a method and check that method in both places. Initially, I was feeling like all the records should pass through the `EmailStolenBikeAlertWorker`, but I don't think that makes sense.